mrrc/mrrc_logger.py

```diff
@@ -0,0 +1,57 @@
+from locale import nl_langinfo, CODESET
+from os import fdopen, dup
+import logging
+import sys
+
+MRRC_INFO_LOGGER_FORMAT='%(asctime)s : %(message)s'
+MRRC_DEBUG_LOGGER_FORMAT='%(asctime)s : %(levelname)s - %(message)s'
+DEFAULT_LOGGER = "mrrc-uploader"
+
+class EncodedStream(object):
+    # The point of this class is to force python to enocde UTF-8
```

Comment:
Why do we need this?

Reply:
I just copy-paste this from osbs as it showed in the comment that they encountered some problems during encoding to console. Maybe this is not needed, I'll remove it.